docs/abi.md

```diff
@@ -138,6 +138,15 @@ functions** as
   - arg 1: Source buffer size in bytes
   - return 0: Channel handle, or zero if not found.
 
+- `node_create: (usize, usize, u64) -> u32`: Create a new Node running the Web
```

Comment:
A [later commit](https://github.com/project-oak/oak/pull/340/commits/1ef4dd32991df27354f5fb7303bfc9cfb2dcff6f) does indeed allow nodes to create logging and storage nodes dynamically, using exactly this host function, so I think it *shouldn't* mention `wasm` or `web_assembly`.

Reply:
Thanks for the pointer, that makes more sense, I'm still a bit confused by the C++ / Rust API though, I would have called the argument `node_configuration_name` or something like that, instead of `contents`?

Reply:
Yeah, "contents" isn't a great term, but it's what's already used in `manager.proto` for this.  Maybe defer and come up with a better name in the follow-up PR?